### DISCUSSION TITLE
Grabaciones clase

### DISCUSSION THREAD

**Discussion by malalalalala** [rramosp/ai4eng.v1#274] [00 - General]
Buenos días.

Hay un problema con el link de las grabaciones de clase https://ingenia.udea.edu.co/zoom/meeting/93614993384, he intentado con varios navegadores y con todos se queda pensando y nunca carga la liusta de videos.
![image](https://github.com/rramosp/ai4eng.v1/assets/59240499/8bb4c2f4-8e53-48fb-a959-e71d497b3eab)

**Comment by Felipe-RA** [comment]
Saludos @lauramosdim 

Si... esto es un problema de la plataforma que aloja las grabaciones, Ingenia.

Esta afectando la mayoría de los cursos de la facultad.

De momento tendremos que esperar a que se normalice la situación

**Reply by Felipe-RA** [reply]
Actulización:

Debido a algunos cambios en la API de Zoom esta fuera el servicio.

El área de IT espera tenerlo arreglado hoy en la tarde, o mañana en la mañana.
